zebra/zebra_evpn_mac.c

```diff
@@ -1642,13 +1757,18 @@ zebra_evpn_proc_sync_mac_update(zebra_evpn_t *zevpn, struct ethaddr *macaddr,
 		inform_bgp = true;
 	}
 
-	if (IS_ZEBRA_DEBUG_EVPN_MH_MAC)
-		zlog_debug("sync-mac %s vni %u mac %s es %s seq %d f 0x%x%s%s",
+	if (IS_ZEBRA_DEBUG_EVPN_MH_MAC) {
+		char mac_buf[MAC_BUF_SIZE];
+
+		zlog_debug("sync-mac %s vni %u mac %s es %s seq %d f %s%s%s",
 			   ctx->mac_created ? "created" : "updated", zevpn->vni,
 			   prefix_mac2str(macaddr, macbuf, sizeof(macbuf)),
 			   mac->es ? mac->es->esi_str : "-", mac->loc_seq,
-			   mac->flags, inform_bgp ? " inform_bgp" : "",
+			   zebra_evpn_zebra_mac_flag_dump(mac, mac_buf,
+							  sizeof(mac_buf)),
+			   inform_bgp ? " inform_bgp" : "",
```

Comment:
I think you can remove the space before inform_bgp here as there is one provided in the flags output.

Reply:
fixed